### PR TITLE
REG-130 fix query

### PR DESCRIPTION
**Pull request by yunhailuo**

### CODE REVIEW

File: src/encoded/regulome_search.py [hunk 1]
```diff
@@ -296,10 +296,9 @@ def get_coordinate(query_term, assembly='GRCh37', atlas=None):
 
 
 def get_rsids(atlas, assembly, chrom, start, end):
-    pos = (int(start) + int(end)) / 2
-    window = int(end) - int(start)
-    rsids = atlas.nearby_snps(_GENOME_TO_ALIAS.get(assembly, 'hg19'),
-                              chrom, pos, window=window, max_snps=99999)
+    rsids = atlas.find_snps(
```
Comment (hitz) [inline src/encoded/regulome_search.py]: Why do you suppose get_rsids was using nearby_snps() in the first place...

Comment (hitz) [reply src/encoded/regulome_search.py]: maybe the old version of find_snps didn't take a range.

Comment (yunhailuo) [reply src/encoded/regulome_search.py]: Could be. I couldn't remember. But I think `find_snps` here is more proper for now.

**Comment by hitz** [comment]
This has conflicts again, probably with 74

**Comment by yunhailuo** [comment]
> This has conflicts again, probably with 74

Right. Working on it now.

**Pull request merged**